relay-server/Dockerfile

```diff
@@ -13,8 +13,29 @@ RUN cd relay-server && make
 
 ### Copy executable image
 
-FROM alpine:3.14
+FROM redhat/ubi9-minimal
 
+ARG VERSION="latest"
+
+LABEL name="kubearmor-relay-server" \
+      vendor="Accuknox" \
```

Comment:
Should we include this? KubeArmor is vendor neutral right?

Reply:
We can go with `AccuKnox`.